sqlalchemy_history/manager.py

```diff
@@ -139,6 +139,9 @@ def reset(self):
         self.declarative_base = None
         self.version_class_map = {}
         self.parent_class_map = {}
+        # Tracke all tables in {assoc_tables mapping, model table mapping}
+        self.version_table_map = {}  # mapping from parent to versioned table
```

Comment:
1. Why is `version_class_map` a dict but `assocication_version_tables` a set ? Can we make it consistent
2. Why do we need `parent_class_map` if we have a `version_class_map` - isnt it just the reversed form ?
3. Why does `association_version_tables` track only the assoc-tables and not all tables ?
  a. We could have a separate variable to track non-assoc tables
  b. Increase scope of the current variable to track both